Mathlib/RingTheory/MvPolynomial/Basic.lean

```diff
@@ -178,6 +178,9 @@ lemma algebraMap_def :
     algebraMap (MvPolynomial σ R) (MvPolynomial σ S) = MvPolynomial.map (algebraMap R S) :=
   rfl
 
+instance : IsScalarTower R (MvPolynomial σ R) (MvPolynomial σ S) :=
```

Comment:
Can you prove more generally that `S` is an `R`-algebra then `IsScalarTower R S (MvPolynomial σ S)`? Since `MvPolynomial.algebraMvPolynomial` is not an instance this cannot be an instance either, but can be a theorem (you can keep this instance of course).

Reply:
Is this not [MvPolynomial.isScalarTower](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/MvPolynomial/Basic.html#MvPolynomial.isScalarTower)?

Reply:
Sorry you're right, something confused me. Thanks!